app/javascript/shared/constants/locales.js

```diff
@@ -0,0 +1,186 @@
+const locales = [
+  { code: 'ab', name: 'Abkhaz', nativeName: 'аҧсуа' },
+  { code: 'aa', name: 'Afar', nativeName: 'Afaraf' },
+  { code: 'af', name: 'Afrikaans', nativeName: 'Afrikaans' },
+  { code: 'ak', name: 'Akan', nativeName: 'Akan' },
+  { code: 'sq', name: 'Albanian', nativeName: 'Shqip' },
+  { code: 'am', name: 'Amharic', nativeName: 'አማርኛ' },
+  { code: 'ar', name: 'Arabic', nativeName: 'العربية' },
+  { code: 'an', name: 'Aragonese', nativeName: 'Aragonés' },
+  { code: 'hy', name: 'Armenian', nativeName: 'Հայերեն' },
+  { code: 'as', name: 'Assamese', nativeName: 'অসমীয়া' },
+  { code: 'av', name: 'Avaric', nativeName: 'авар мацӀ, магӀарул мацӀ' },
+  { code: 'ae', name: 'Avestan', nativeName: 'avesta' },
+  { code: 'ay', name: 'Aymara', nativeName: 'aymar aru' },
+  { code: 'az', name: 'Azerbaijani', nativeName: 'azərbaycan dili' },
+  { code: 'bm', name: 'Bambara', nativeName: 'bamanankan' },
+  { code: 'ba', name: 'Bashkir', nativeName: 'башҡорт теле' },
+  { code: 'eu', name: 'Basque', nativeName: 'euskara, euskera' },
+  { code: 'be', name: 'Belarusian', nativeName: 'Беларуская' },
+  { code: 'bn', name: 'Bengali', nativeName: 'বাংলা' },
+  { code: 'bh', name: 'Bihari', nativeName: 'भोजपुरी' },
+  { code: 'bi', name: 'Bislama', nativeName: 'Bislama' },
+  { code: 'bs', name: 'Bosnian', nativeName: 'bosanski jezik' },
+  { code: 'br', name: 'Breton', nativeName: 'brezhoneg' },
+  { code: 'bg', name: 'Bulgarian', nativeName: 'български език' },
+  { code: 'my', name: 'Burmese', nativeName: 'ဗမာစာ' },
+  { code: 'ca', name: 'Catalan', nativeName: 'Català' },
+  { code: 'ch', name: 'Chamorro', nativeName: 'Chamoru' },
+  { code: 'ce', name: 'Chechen', nativeName: 'нохчийн мотт' },
+  { code: 'zh', name: 'Chinese', nativeName: '中文 (Zhōngwén), 汉语, 漢語' },
+  { code: 'cv', name: 'Chuvash', nativeName: 'чӑваш чӗлхи' },
+  { code: 'kw', name: 'Cornish', nativeName: 'Kernewek' },
+  { code: 'co', name: 'Corsican', nativeName: 'corsu, lingua corsa' },
+  { code: 'cr', name: 'Cree', nativeName: 'ᓀᐦᐃᔭᐍᐏᐣ' },
+  { code: 'hr', name: 'Croatian', nativeName: 'hrvatski' },
+  { code: 'cs', name: 'Czech', nativeName: 'česky, čeština' },
+  { code: 'da', name: 'Danish', nativeName: 'dansk' },
+  { code: 'dv', name: 'Divehi; Dhivehi; Maldivian;', nativeName: 'ދިވެހި' },
+  { code: 'nl', name: 'Dutch', nativeName: 'Nederlands, Vlaams' },
+  { code: 'en', name: 'English', nativeName: 'English' },
```

Comment:
Can we have variants as well? (en-us, en-GB)

Reply:
Sure

Reply:
@pranavrajs I am planning to use all locales from this list https://github.com/umpirsky/locale-list/blob/master/data/en/locales.json